pkg/base1/cockpit.js

```diff
@@ -798,7 +798,7 @@ function Channel(options) {
         self.valid = false;
         if (transport && id)
             transport.unregister(id);
-        if (closed.message)
+        if (closed.message && options.err !== "message")
```

Comment:
This one I am not super sure about - can it possibly hide some messages we might need?

Reply:
That's a good point -- it's [documented](https://cockpit-project.org/guide/latest/cockpit-spawn.html) to return the error through the API and *not* the console, and we specifically use `message` in many cases to *avoid* it going to the journal or console, and just ignore them; the latter should really use `err: "ignore"`, but I figure we are not terribly consistent about it.

But either way, this would still mean that the console would get closed messages for `err` == `"ignore"` or `"out"`, which we also don't want. So it almost seems to me that this should be `&& !options.err` ?